### PR TITLE
5.37 upgrade notes

### PR DESCRIPTION
**Pull request by amyblais**
Upgrade notes will also be added to changelog and sent to Gitlab.

### CODE REVIEW

**Review by cwarnermm — APPROVED** [review]

**Review by itao — APPROVED** [review]

File: source/administration/important-upgrade-notes.rst [hunk 1]
```diff
@@ -2,13 +2,25 @@ Important Upgrade Notes
...
+|                                                    | v5.37 and later. It’s expected that you may experience bugs as we stabilize the feature. In particular, please be aware of                                       |
+|                                                    | `the known issues documented here <https://docs.mattermost.com/help/messaging/organizing-conversations.html#known-issues>`_.                                     |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | v5.37 adds support for emoji standard v13.0. If you have added a custom emoji in the past that uses one of the new system names, then it is going to get         |
+|                                                    | overwritten by the system emoji. The workaround is to change the custom emoji name.                                                                              |
```
Comment (Willyfrog) [inline source/administration/important-upgrade-notes.rst:19]: maybe mention that the emoji picker now supports selecting a default skin tone?

**Review by Willyfrog — APPROVED** [review]
it looks good, added a comment in case you want to mention that as well (I think there was a client requesting that feature)

File: source/administration/important-upgrade-notes.rst (hunk 1 above)
Comment (amyblais) [reply source/administration/important-upgrade-notes.rst:19]: The changelog draft mentions it, I think https://github.com/mattermost/docs/pull/4702/files

Comment (Willyfrog) [reply source/administration/important-upgrade-notes.rst:19]: ah! great then :) disregard my comment

**Review by esethna — APPROVED** [review]

File: source/administration/important-upgrade-notes.rst [hunk 2]
```diff
@@ -2,13 +2,25 @@ Important Upgrade Notes
...
-   - The ``platform`` binary and “--platform” flag will be deprecated in a future release. If you are using the “--platform” flag or are using the ``platform`` binary directly to run the Mattermost server application via a systemd file or custom script, you will be required to use only the ``mattermost`` binary.
 
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v5.37.0                                            | The ``platform`` binary and “--platform” flag will be deprecated in a future release. If you are using the “--platform” flag or are using the ``platform`` binary|
```
Comment (isacikgoz) [inline source/administration/important-upgrade-notes.rst:11]:
```suggestion
| v5.37.0                                            | The ``platform`` binary and “--platform” flag has been removed. If you are using the “--platform” flag or are using the ``platform`` binary|
```

**Review by isacikgoz — COMMENTED** [review]
We already removed the platform binary and flags, I think we should change the wording below:

File: source/administration/important-upgrade-notes.rst (hunk 2 above)
Comment (amyblais) [reply source/administration/important-upgrade-notes.rst]: My bad, fixed.

**Review by isacikgoz — APPROVED** [review]
LGTM 👍

**Review by wiersgallak — APPROVED** [review]

**Pull request merged**